### ISSUE TITLE
not able to create a client when lxd is installed from snap on artful

### ISSUE THREAD

**Issue by codersquid** [lxc/pylxd#257]
I discovered this while trying to use mojo installed from a snap. I have a new laptop where I've installed lxd from a snap. On my other machine, where lxd is installed from a deb, I can instantiate a client fine. But on this one, instantiation fails.

```
» python              
Python 2.7.14 (default, Sep 23 2017, 22:06:14) 
[GCC 7.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import pylxd
>>> pylxd.Client()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/codersquid/work/mojo/trunk/vpylxd/local/lib/python2.7/site-packages/pylxd/client.py", line 232, in __init__
    raise exceptions.ClientConnectionFailed()
pylxd.exceptions.ClientConnectionFailed
>>> 
```

If I don't have a chance to investigate this, this section is where things were failing
https://github.com/lxc/pylxd/blob/master/pylxd/client.py#L246

**Comment by ajkavanagh** [comment]
Hi @codersquid: This issue was fixed in PR: #250, which was on Sep 26, 2017.  Unfortunately, this isn't in a released version yet (last release was 2.2.4 - July 10, 2017).  I guess the snap version of lxd is gaining traction.

If you need it now then git clone and "pip install -e ." will give you the current master version.

We should probably roll out a new release very soon.  Leaving open as a reminder.

**Comment by shuhaowu** [comment]
any idea when a new version will be released?

**Comment by ajkavanagh** [comment]
This has now been fixed in 2.2.5 which has now been release.
